app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module SignUp
+  class CompletionsController < ApplicationController
+    def show
+      analytics.track_event(
+        Analytics::USER_REGISTRATION_AGENCY_HANDOFF_PAGE_VISIT,
+        service_provider_attributes
+      )
+    end
+
+    def update
+      analytics.track_event(
+        Analytics::USER_REGISTRATION_AGENCY_HANDOFF_COMPLETE,
+        service_provider_attributes
+      )
+      redirect_to session[:saml_request_url]
+    end
+
+    private
+
+    def service_provider_attributes
+      { loa3: session[:sp].try(:loa3), service_provider_name: @sp_name }
```

Comment:
`try` is required in that we do not require there to be an `sp` session var in order to get on this page. So in the specs, we do not have that set up. Should we enforce the existence of that session var before letting a user access these endpoints?